style.css

```diff
@@ -0,0 +1,103 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap');
+
+* {
+  padding: 0;
+  margin: 0;
+  font-family: "Poppins", sans-serif;
+  box-sizing: border-box;
+}
+
+body {
+  width: 375px;
+  height: 718px;
+  margin: auto;
+}
+
+nav {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 58px 24px;
+  text-decoration: none;
+  color: #6070ff;
+  width: 375px;
+}
+
+nav a {
+  text-decoration: none;
+}
+
+nav a:hover {
+  background-color: rgb(124, 118, 83);
+  transform: scale(1.5);
+}
+
+.logo {
+  color: #6070ff;
+  width: 76px;
+  height: 20px;
+  font-size: 18px;
+  font-family: "Poppins", sans-serif;
+}
+
+main {
+  background-image: url('Images/header-shapes\ mobile@2x.png');
```

Comment:
As highlighted by the previous reviewer, I suggest you implement the background images to be visible in your page
this is your page currently:

<img width="1610" alt="Screenshot 2022-10-27 at 11 23 40" src="https://user-images.githubusercontent.com/48546992/198232499-b40138e2-90e5-4e08-a65d-07c8380895c7.png">
As you can see, no background images are visible

This is the figma template for your reference
<img width="374" alt="Screenshot 2022-10-27 at 11 25 49" src="https://user-images.githubusercontent.com/48546992/198232683-7ac97de3-106e-4d81-9f7c-faf9342b06c2.png">